src/connectors/github_organization.py

```diff
@@ -0,0 +1,339 @@
+        'required': True,
+    }
+]
+
+FILE_FORMAT = """
+    TYPE = "JSON",
+    COMPRESSION = "AUTO",
+    ENABLE_OCTAL = FALSE,
+    ALLOW_DUPLICATE = FALSE,
+    STRIP_OUTER_ARRAY = TRUE,
+    STRIP_NULL_VALUES = FALSE,
+    IGNORE_UTF8_ERRORS = FALSE,
+    SKIP_BYTE_ORDER_MARK = TRUE
+"""
+
+LANDING_TABLE_COLUMNS = [
+    ('insert_time', 'TIMESTAMP_LTZ(9)'),
+    ('raw', 'VARIANT'),
+    ('hash_raw', 'NUMBER'),
+    ('ref', 'VARCHAR(256)'),
+    ('before', 'VARCHAR(256)'),
+    ('after', 'VARCHAR(256)'),
+    ('created', 'BOOLEAN'),
+    ('deleted', 'BOOLEAN'),
+    ('forced', 'BOOLEAN'),
+    ('base_ref', 'VARCHAR(256)'),
+    ('compare', 'VARCHAR(256)'),
+    ('commits', 'VARIANT'),
+    ('head_commit', 'VARIANT'),
+    ('repository', 'VARIANT'),
+    ('pusher', 'VARIANT'),
+    ('organization', 'VARIANT'),
+    ('sender', 'VARIANT'),
+    ('action', 'VARCHAR(256)'),
+    ('check_run', 'VARIANT'),
+    ('check_suite', 'VARIANT'),
+    ('number', 'NUMBER(38,0)'),
+    ('pull_request', 'VARIANT'),
+    ('label', 'VARIANT'),
+    ('requested_team', 'VARIANT'),
+    ('ref_type', 'VARCHAR(256)'),
+    ('master_branch', 'VARCHAR(256)'),
+    ('description', 'VARCHAR(256)'),
+    ('pusher_type', 'VARCHAR(256)'),
+    ('review', 'VARIANT'),
+    ('changes', 'VARIANT'),
+    ('comment', 'VARIANT'),
+    ('issue', 'VARIANT'),
+    ('id', 'NUMBER(38,0)'),
+    ('sha', 'VARCHAR(256)'),
+    ('name', 'VARCHAR(256)'),
+    ('target_url', 'VARCHAR(8192)'),
+    ('context', 'VARCHAR(256)'),
+    ('state', 'VARCHAR(256)'),
+    ('commit', 'VARIANT'),
+    ('branches', 'VARIANT'),
+    ('created_at', 'TIMESTAMP_LTZ(9)'),
+    ('updated_at', 'TIMESTAMP_LTZ(9)'),
+    ('assignee', 'VARIANT'),
+    ('release', 'VARIANT'),
+    ('membership', 'VARIANT'),
+    ('alert', 'VARIANT'),
+    ('scope', 'VARCHAR(256)'),
+    ('member', 'VARIANT'),
+    ('requested_reviewer', 'VARIANT'),
+    ('team', 'VARIANT'),
+    ('starred_at', 'TIMESTAMP_LTZ(9)'),
+    ('pages', 'VARIANT'),
+    ('project_card', 'VARIANT'),
+    ('build', 'VARIANT'),
+    ('deployment_status', 'VARIANT'),
+    ('deployment', 'VARIANT'),
+    ('forkee', 'VARIANT'),
+    ('milestone', 'VARIANT'),
+    ('key', 'VARIANT'),
+    ('project_column', 'VARIANT'),
+    ('status', 'VARCHAR(256)'),
+    ('avatar_url', 'VARCHAR(256)')
+]
+
+CONNECT_RESPONSE_MESSAGE = """
+STEP 1: Modify the Role "{role}" to include the following trust relationship:
+
+{role_trust_relationship}
+
+
+STEP 2: For Role "{role}", add the following inline policy:
+
+{role_policy}
+"""
+
+
+def connect(connection_name, options):
+    base_name = f'GITHUB_ORGANIZATION_{connection_name}_EVENTS'.upper()
+    stage = f'data.{base_name}_STAGE'
+    staging_table = f'data.{base_name}_STAGING'
+    landing_table = f'data.{base_name}_CONNECTION'
+
+    bucket = options['bucket_name']
+    prefix = options.get('filter', S3_BUCKET_DEFAULT_PREFIX)
+    role = options['aws_role']
+
+    comment = f"""
```

Comment:
I know some of our older code still has this, but please use `utils.yaml_dump(...)`